.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: python
-sudo: false
 python:
-  - "2.7"
-  - "3.5"
   - "3.6"
```

Comment:
meaningfulconsent is still on `point`, so when run in python 3 it would use python 3.4. I would add python 3.4 to travis here to make sure that works.